pkg/manager/member/tidb_member_manager.go

```diff
@@ -241,10 +244,87 @@ func (m *tidbMemberManager) syncTiDBStatefulSetForTidbCluster(tc *v1alpha1.TidbC
 			return err
 		}
 	}
+	// set random password
+	if tc.Spec.TiDB.Initializer != nil && tc.Spec.TiDB.Initializer.CreatePassword && !tc.Status.TiDB.InitPassword {
```

Comment:
Is it better we define a `phase` in the status as we do for the initializer here https://github.com/pingcap/tidb-operator/blob/master/pkg/apis/pingcap/v1alpha1/tidbinitializer_types.go#L109 so that we can know if the password creation is ongoing, successful, or failed?
The bool `InitPassword` cannot represent all the statuses.

Reply:
ok,i use same type as the phase field in tidbinitializer.